tracer/src/Datadog.Trace/RemoteConfigurationManagement/RcmCapabilitiesIndices.cs

```diff
@@ -34,6 +34,9 @@ internal static class RcmCapabilitiesIndices
         public const uint AsmUserBlockingUInt32 = 1 << 7;
         public static readonly BigInteger AsmUserBlocking = new(AsmUserBlockingUInt32);
 
+        public const uint AsmCustomRulesUInt32 = 1 << 8;
+        public static readonly BigInteger AsmCustomRules = new(AsmUserBlockingUInt32);
+
```

Comment:
Is this ok? Should we make a new with AsmCustomRulesUInt32 instead of AsmUserBlockingUInt32?

Reply:
thank you!